Makefile

```diff
@@ -77,6 +77,8 @@ run-collector: clone-tnf-secrets stop-running-collector-container
 		-e SERVER_ADDR=':${HOST_PORT}' \
 		-e SERVER_READ_TIMEOUT=10 \
 		-e SERVER_WRITE_TIMEOUT=10 \
+		-e AWS_ACCESS_KEY=$(shell jq -r ".CollectorAWSAccessKey" "./tnf-secrets/collector-secrets.json") \
+		-e AWS_SECRET_ACCESS_KEY=$(shell jq -r ".CollectorAWSSecretAccessKey" "./tnf-secrets/collector-secrets.json") \
```

Comment:
I think it will work without `""`.

Reply:
yes your'e right, though is that an important fix?